openedx/features/course_experience/views/course_outline.py

```diff
@@ -14,18 +14,25 @@
 from web_fragments.fragment import Fragment
 
 from courseware.courses import get_course_overview_with_access
+from courseware.model_data import FieldDataCache
```

Comment:
can it be removed? since we don't use any of it

Reply:
we use it